### PR TITLE
Setupdocs

### PR DESCRIPTION
**Pull request by ankitmashu**

### CODE REVIEW

**Comment by kailash** [comment]
ok to test

**Review by kailash — CHANGES_REQUESTED** [review]
Check preview of your document [here](https://github.com/ankitmashu/iudx-gis-interface/blob/setupdocs/SETUP.md), 

1. duplicate `setting up RMQ` heading
2. incorrectly formatted Databroker Verticle config

**Comment by ankitmashu** [comment]
> Check preview of your document [here](https://github.com/ankitmashu/iudx-gis-interface/blob/setupdocs/SETUP.md),
> 
> 1. duplicate `setting up RMQ` heading
> 2. incorrectly formatted Databroker Verticle config

Solved issue

File: SETUP.md [hunk 1]
```diff
@@ -119,48 +75,23 @@ CREATE TABLE IF NOT EXISTS gis
 ```
 ----
 
-## Setting up ImmuDB for IUDX Gis Interface
-- Refer to the docker files available [here](https://github.com/datakaveri/iudx-deployment/blob/master/Docker-Swarm-deployment/single-node/immudb) to setup ImmuDB.
-- Refer [this](https://github.com/datakaveri/iudx-deployment/blob/master/Docker-Swarm-deployment/single-node/immudb/docker/immudb-config-generator/immudb-config-generator.py) to create table/user.
-- In order to connect to the appropriate ImmuDB database, required information such as meteringDatabaseIP, meteringDatabasePort etc. should be updated in the MeteringVerticle module available in [config-example.json](configs/config-example.json).
+## Setting up Metering for IUDX Gis Interface
+- In Metering Verticle we are pushing data in Auditing Verticle through RabbitMQ. [config-example.json](configs/config-example.json).
```
Comment (kailash) [inline SETUP.md:79]:
```suggestion
- In Metering Verticle we are pushing data in Auditing server through RabbitMQ. [config-example.json](configs/config-example.json).
```

**Pull request merged**